docs/logs/fields.md

```diff
@@ -4,4 +4,68 @@ description: Sematext Logs App looks for special fields in logs, namely os.host,
 Every log event shipped to Sematext Logs has its structure - it is divided into fields. Each field has a [type](/logs/field-types/), for example [string](/logs/field-types/#string), [date](/logs/field-types/#date), or [integer](/logs/field-types/#integerlong). It can even be an object holding structured data.  We do everything we can to ensure that log event field types are inferred correctly. However, you may also want to set field types explicitly. This can be done using the Field Editor accessible via a Logs App settings or using the [Templates and Mappings](/logs/mappings-templates) APIs.
 
 ## Common Schema
-Fields in log events are also referred to as Tags in Sematext.  They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+Fields in log events are also referred to as Tags in Sematext. They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+
+## Fields Structure
+
+The structure of your data is dictated by the data shipped to Sematext Logs as log events. There are two places where you can easily see your data structure - the fields and filters section of your Logs table and the [Field Editor](/logs/fields/#field-editor):
+
+<img src="../../images/logs/logs_structure_fields_and_filters.png" alt="Logs Fields and Filters">
+
+<img src="../../images/logs/logs_structure_field_editor.png " alt="Logs Field Editor">
+
+---
+**Note:**
+The structure shown by the fields and filters section and the field editor may differ. The data structure shown by the fields and filters always covers the **whole data** - old and new. The data structure shown by the Field Editor is always the **current** one, the one that will be applied to all your current and future events.
+--- 
+
+### Field Types
+
+Each field has a field type associated with it and will include:
```

Comment:
> and will include

Not sure why we need this?  Maybe we just want to say this:
Each field has a field type. The following field types are supported:

Reply:
Changed.